src/extension/background-script/actions/accounts/export.ts

```diff
@@ -0,0 +1,32 @@
+import { decryptData } from "~/common/lib/crypto";
+import state from "~/extension/background-script/state";
+import type { MessageAccountExport } from "~/types";
+
+const exportAccount = async (message: MessageAccountExport) => {
+  const accounts = state.getState().accounts;
+  const password = state.getState().password as string;
+  const accountId = message.args.id;
+
+  if (accountId in accounts) {
+    if (accounts[accountId].connector === "lndhub") {
+      const lndHubData = decryptData(
+        accounts[accountId].config as string,
+        password
+      );
+
+      return {
+        data: lndHubData,
+      };
+    } else {
+      return {
+        error: `Account: ${accountId} not an LndHub Account; cannot be exported`,
```

Comment:
theoretically we could do this for all accounts. it is just returning the decrypted config. 

we can only do the check for the lndhub on the frontend. later we then could also display the details of other connector types. 

what do you think?

Reply:
Yeah, I added this while adding tests, this is checked on the frontend only. Will change!